tests/unit/Params/UserParamsTest.php

```diff
@@ -50,4 +50,11 @@ public function testInvalidOrcidChecksum(): void
         $this->expectException(ImproperActionException::class);
         $params->getContent();
     }
+
+    public function testClearOrcid(): void
```

Comment:
in `testOrcid()` function, you can add an empty string and assert that the saved orcid is now null